src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -52,14 +52,13 @@
 TOOLS_AVAILABLE = ['gtdb_tk', 'checkm2', 'microtrait', 'mash', 'eggnog']
 
 # estimated execution time (in minutes) for each tool to process a chunk of data
-TASK_META = {'gtdb_tk': {'chunk_size': 1000, 'exe_time': 65},
-             'eggnog': {'chunk_size': 1000, 'exe_time': 65},  # TODO: update this value after performance testing
+TASK_META = {'gtdb_tk': {'chunk_size': 1000, 'exe_time': 65, 'tasks_per_node': 4},
+             'eggnog': {'chunk_size': 100, 'exe_time': 15, 'node_time_limit': 0.5},  # Memory intensive tool - reserve more nodes with less node reservation time
```

Comment:
does this make it harder to reserve a set of nodes, as my understanding is that they all have to be available simultaneously as a block?

Reply:
that's true. So we might have to come back and update this again once we have run large data sets.

Reply:
Yeah, we might need to make it submit several jobs with their own blocks

Reply:
:+1: